docs/source/en/api/pipelines/stable_diffusion/overview.md

```diff
@@ -172,3 +172,26 @@ inpaint = StableDiffusionInpaintPipeline(**text2img.components)
 
 # now you can use text2img(...), img2img(...), inpaint(...) just like the call methods of each respective pipeline
 ```
+
+### Create web demos using `gradio`
+
+The Stable Diffusion pipelines are automatically supported in [the gradio library](https://github.com/gradio-app/gradio/), making it a breeze to create a web demo
+based on your pipeline. First, install `gradio` with: `pip install gradio`.
+
+Then, create a web demo around an image generation pipeline (or any other Stable Diffusion pipeline) in a single line of code:
```

Comment:
I think it might be better to rephrase it like this so users know explicitly they can use other Stable Diffusion-based pipelines for other tasks like image-to-image and inpainting.

Maybe even include a code example of launching a Gradio app from an image-to-image pipeline :)

```suggestion
Then, create a web demo around any Stable Diffusion-based pipeline. For example, to create an image generation pipeline in a single line of code:
```